lib/pry/config.rb

```diff
@@ -4,6 +4,158 @@ class Pry
   class Config < Pry::BasicObject
     include Behavior
 
+    # rubocop:disable Metrics/MethodLength
+    def self.defaults
+      defaults = from_hash(
```

Comment:
`defaults` is assigned here because some configuration options read from other configuration options to return a value. basically allows one config option to call another, like `defaults.other_option`.